docs/filter-api.md

```diff
@@ -6,16 +6,18 @@ permalink: /docs/filter-api/
 
 ##Dust Filters
 ###Function definition
-    
-    dust.helpers.myFilter = function(value) {
-      /* Modify the value */
-      return value; 
-    }
-    
+```    
+dust.filters.myFilter = function(value) {
+  /* Modify the value */
+  return value; 
+}
+``` 
 ###Parameters
 * value: The value that is used in the dust reference.
```

Comment:
Can you be more specific about what "the value that is used in the Dust reference" means?